packages/terra-tabs/src/common-tabs/_Tabs.jsx

```diff
@@ -143,6 +179,19 @@ class Tabs extends React.Component {
         return { visibleTabData: newArray };
       });
     }
+
+    if (currTab && this.addButtonToggle && !this.moreButtonRef.current) {
+      const element = document.getElementById(currTab.id);
+      if (element && this.addButtonToggle) {
+        element.focus();
+      }
+    }
+
+    const element = document.getElementById(this.addTabId);
+    if (this.state.visibleTabData.length === 0 && element) {
+      element.focus();
+    }
```

Comment:
```suggestion
    const element = document.getElementById(this.addTabId);
    if (this.state.visibleTabData.length === 0 && element) {
      element.focus();
    }
```
I believe the above block should be a else part of above condition line no: 185 `if (element && this.addButtonToggle) {`

Reply:
Done https://github.com/cerner/terra-framework/commit/628052153b88461ef88733be6b05c1fb519e3f41